docs/guide/migrating-to-version-9.md

```diff
@@ -548,6 +548,7 @@ This component has been removed. Please refer to KBadge component which has been
 
 ### KModalFullscreen
 
+This component is deprecated and will be removed in the next major release.
```

Comment:
Remind me here why we aren't removing it now? Is it still used in Konnect?

Reply:
Yes - 7 occurrences in `konnect-ui-apps`, 4 more on `khcp-ui`.